governance/votes/Executive Vote - April 4, 2024.md

```diff
@@ -0,0 +1,92 @@
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) Increase
+
+- **Authorization:** [Poll](https://vote.makerdao.com/polling/QmcLsYwj)  
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/gsm-pause-delay-increase-proposal/23929)
+
+If this executive proposal passes, The Governance Pause Delay (GSM) will be increased by 14 hours, from 16 to **30 hours**.
+
+### Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Increase
+
+- **Authorization:** [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006/9)  
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)
+
+If this executive proposal passes, the Spark MetaMorpho Vault DDM Maximum Debt Ceiling (line) will increase by 900 million DAI, from 100 million to **1 billion DAI**.
+
+### Add ilks to [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker)
+
+- **Authorization:** [Poll](https://vote.makerdao.com/polling/QmZsAM36#vote-breakdown)  
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/gov12-1-2-bootstrapping-edit-proposal-gov10-2-3-1a-edit/24005)
+
+If this executive proposal passes, the following ilks will be added to the Debt Ceiling Breaker (LINE_MOM):
+
+* ETH-A
+* ETH-B
+* ETH-C
+* WSTETH-A
+* WSTETH-B
+* WBTC-A
+* WBTC-B
+* WBTC-C
+
+### Trigger SparkLend Proxy Spell
+
+- **Authorization:** Polls: [1](https://vote.makerdao.com/polling/QmdjqTvL), [2](https://vote.makerdao.com/polling/QmaEqEav), [3](https://vote.makerdao.com/polling/QmbCWUAP). [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)  
+- **Proposal:** Forum posts: [1](https://forum.makerdao.com/t/mar-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/23918), [2](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)
+
+If this executive proposal passes, the SparkLend Proxy spell [0x7748C5E6EEda836247F2AfCd5a7c0dA3c5de9Da2](https://etherscan.io/address/0x7748c5e6eeda836247f2afcd5a7c0da3c5de9da2) will be triggered. In addition to polled changes on Activating the Kill Switch Security Module and increasing WBTC and ETH lending caps, this proxy spell contains changes suggested and approved in this [forum post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006).
```

Comment:
:yellow_circle: **Medium**

We strongly advise that Spark spell items are itemized for the benefit of voters like in previous docs.

See: https://github.com/makerdao/community/pull/2597/files#diff-46eac166654845e1c858fca919ba2d2ec7d5114fdc7248747924e2fc3b8b8e14R125-R127
and: https://github.com/makerdao/community/pull/2580/files#diff-a856ab68877ffd3cd18fa55b98aee86bfd37a7f0c1b7ce54bee1e43ee9634d00R102-R225